Sources/ISDBTestSupport/ForwardingIndexDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```

Comment:
`2020` for new files 😉

Reply:
I did the indexstore-db side of this in December. I just sat on the changes until I could test them thoroughly from the sourcekit-lsp side.  I can still update the dates if you think it's correct :shrug:  Wish we could just remove the dates from the headers.  No one is consistent about keeping them updated in existing files.

Reply:
It's fine, I don't really care much.